project-icp/bee-pollinator-app
# Small demo improvements

#### Overview
Some miscellaneous fixes that added small polish to a recent demo:
* Fix syntax error for django auth backend which prevented user from logging in
* Show an overlay layer from Crop (using NLCD)
    * Rename code and files to more clearly represent that the control toggles an overlay, not an opacity slider.
    * Use a temporary overlay (NLCD) while the crop layer is being developed
    * Update the code to remove redundancies, read from configuration and have updated text
* Update text label for Managed Hives control from `Value` to `Managed Hives per Acre`

Connects #64 

#### Testing
1. Use the crop data toggle button
    * Should show the NLCD
    * Updated text to "Crop Layer" from "CDL Layer"
    * Added tooltip which explains the action clicking will toggle (show/hide)
2. Create a user `./scripts/manage.sh createsuperuser`
    * Log in to the app using the above credentials
3. Verify text for "Value" has been replaced with more meaningful text

![imageedit_4_7795560672](https://cloud.githubusercontent.com/assets/1014341/20852932/cf6b8722-b8b6-11e6-9a28-905c99742693.gif)
